specs/core/1_custody-game.md

```diff
@@ -131,7 +131,7 @@ The following types are defined, mapping into `DomainType` (little endian):
 
 | Name | Value |
 | - | - |
-| `DOMAIN_CUSTODY_BIT_CHALLENGE` | `6` |
+| `DOMAIN_CUSTODY_BIT_CHALLENGE` | `0x05000000` |
```

Comment:
Careful here. 5 used to be the transfer domain type. Hope clients pick up on this change. Some may still have the unused transfer code around.

Reply:
I am aware, but I did the switch so as not to have an empty index. Happy to skip `5` if there are concerns about clients missing the change.

Reply:
An alternate idea would be to put the change from `uints` -> `bytes` in a separate PR which highlights the changes.

Reply:
I've reverted this change for the aforementioned clarity reasons. I had failed to make the changes in the configs.yaml which was further evidence for your point.